Package.resolved

```diff
@@ -41,8 +41,8 @@
         "package": "SymbolKit",
         "repositoryURL": "https://github.com/apple/swift-docc-symbolkit",
         "state": {
-          "branch": "main",
-          "revision": "ed8ce5502e563090ab1400b4dd7d703b01eceabb",
+          "branch": "revert-24-revert-21-objc-kinds",
```

Comment:
I guess we can land the SymbolKit PR first and re-run CI here since its backwards compatible?